### PR TITLE
feat: add resilience policy boundaries

### PR DESCRIPTION
**Pull request by flyingrobots**
## Summary

- codifies the v0.0.6 resilience boundary: Rust uses `ninelives`, JavaScript tooling uses `@git-stunts/alfred`
- adds `ResiliencePolicy` and `ResilientLoweringPort` for opt-in Rust lowering timeouts without wrapping deterministic compiler errors
- extracts a shared Alfred child-process runner and routes parity/performance native probes through bounded timeout and buffer controls
- updates BEARING, design packet `0015`, CHANGELOG, scripts docs, and `wesley-core` docs

## Validation

- `cargo test -p wesley-core`
- `cargo test -p wesley-core --test resilience_policy`
- `BATS_LIB_PATH=packages/wesley-cli/test bats -t test/ir-parity-sentinel.bats test/ir-performance-baseline.bats`
- `pnpm parity:ir`
- `pnpm perf:ir -- --list-fixtures`
- `pnpm run preflight`
- `pnpm run lint`
- `pnpm run format:check`
- `git diff --check`
- pre-push hook: repository preflight, repo Bats smoke suite, `@wesley/core` tests

## Notes

This PR also includes two formatter-only cleanups in existing files so `pnpm run format:check` is green for the branch.

### CODE REVIEW

**Comment by flyingrobots** [comment]
## Review-Loop Status

| Gate | Status | Notes |
| --- | --- | --- |
| Draft state | Open | PR is now ready for review. |
| CI/checks | Green | `gh pr checks 522` reports all required jobs passing; website deploy is skipped as expected. |
| Review threads | Clear | Thread-aware GraphQL read found 0 review threads. |
| CodeRabbit | Waiting | CodeRabbit attempted review but hit review/credit rate limits. Cooldown comment currently indicates refill around 2026-05-23 16:39:25 PDT. |
| Local audit | Clean | No self-discovered code issues requiring a follow-up commit. |
| Local validation | Green | Re-ran `pnpm run preflight && pnpm run lint && pnpm run format:check`; also ran `cargo clippy -p wesley-core --all-targets -- -D warnings`. |

Merge gate remains review-blocked until a real review arrives or an explicit admin override is chosen.

**Comment by flyingrobots** [comment]
@codex Self-Code Review findings for PR #522. Please confirm the severity and mitigation.

| Severity | File / Lines | Type | Finding | Evidence | Recommended mitigation prompt |
| --- | --- | --- | --- | --- | --- |
| P2 Major | `crates/wesley-core/src/resilience.rs:102-110`; supporting context `crates/wesley-core/src/adapters/apollo.rs:37-39` | Behavioral correctness / overclaimed resilience semantics | `ResilientLoweringPort` exposes a schema-lowering timeout, but the current in-process Apollo lowerer performs synchronous `parse_and_lower` work inside an `async fn`. `tokio::time::timeout` / `ninelives::TimeoutPolicy` can only observe cancellation when the wrapped future yields; it cannot preempt CPU-bound synchronous parser/lowering work. This means callers can reasonably believe they have a hard lowering deadline when common lowerer work may run past the configured timeout. | The wrapper applies `TimeoutPolicy::execute` around `self.inner.lower_sdl(sdl).await`; `ApolloLoweringAdapter::lower_sdl` immediately calls `self.parse_and_lower(sdl)` without an await or process/task boundary. Existing tests prove a pending cooperative future times out, but do not prove hard deadlines for synchronous lowering. | `Prompt: Tighten the Rust resilience boundary so timeout semantics are not overclaimed. Either rename/document ResilientLoweringPort as a cooperative async timeout wrapper and add tests/docs that state CPU-bound in-process lowerers are not preempted, or introduce a separate hard-boundary execution path for lowerers that must be forcibly bounded. Update CHANGELOG/BEARING/design text accordingly so v0.0.6 does not imply hard in-process parser deadlines.` |

Issue count: P0 0, P1 0, P2 1, P3 0, P4/P5 0.

No unresolved PR review threads were present when this self-review ran; this is a self-discovered issue from local diff review against `origin/main`.

**Comment by flyingrobots** [comment]
@codex Self-review issue resolved.

| Issue | Severity | Resolution | Commit | Validation |
| --- | --- | --- | --- | --- |
| Rust lowering timeout semantics overclaimed hard preemption for synchronous in-process work | P2 Major | Renamed the Rust API to `cooperative_lowering_timeout`, documented that the wrapper observes async cancellation points rather than preempting CPU-bound parser/lowering polls, added a synchronous-poll regression test, and updated BEARING/design/README/CHANGELOG wording. | `26d47e2b` | `cargo test -p wesley-core`; `cargo test -p wesley-core --test resilience_policy`; `cargo clippy -p wesley-core --all-targets -- -D warnings`; `pnpm run preflight`; `pnpm run lint`; `pnpm run format:check`; `git diff --check`; push-time repository preflight |

File: crates/wesley-core/src/resilience.rs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            return self.inner.lower_sdl(sdl).await;
+        };
+
+        let policy = TimeoutPolicy::new(timeout)
+            .map_err(|error| WesleyError::ResilienceError(format!("invalid timeout: {error}")))?;
+        let result: Result<WesleyIR, NineLivesError<WesleyError>> = policy
+            .execute(|| async {
+                self.inner
+                    .lower_sdl(sdl)
+                    .await
```
[the remark this answers is not in the document]
Comment (flyingrobots) [reply crates/wesley-core/src/resilience.rs:127]: Resolved in 26d47e2b. The Rust API now names this as cooperative timeout policy (`cooperative_lowering_timeout`), docs state that in-process parser/lowering work is not hard-preempted, and `resilient_lowering_port_does_not_preempt_synchronous_poll_work` pins the limitation in tests.

[the remark this answers is not in the document]
Comment (flyingrobots) [reply crates/wesley-core/src/resilience.rs:127]: Resolved in 26d47e2b. Chose the explicit-documentation/API path: the wrapper is now a cooperative async timeout boundary, not a hard deadline for synchronous CPU-bound lowering, and the test suite includes a synchronous-poll guard.
